src/Hng.Web/appsettings.json

```diff
@@ -36,9 +36,9 @@
     "ApiSecret": ""
   },
   "EmailTemplateDirectory": {
-    "path": ""
+    "path": "../Hng.Infrastructure/EmailTemplates"
```

Comment:
Remove appsettings secretes

Reply:
I pulled this appsettings this dev. Will adjust it anyway but ask the person that initially pushed this to make necessary adjustments on his part too.